cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/GetBrokerStatsRunnable.java

```diff
@@ -22,6 +22,7 @@
   private final ModelCompletenessRequirements _modelCompletenessRequirements;
   private final boolean _allowCapacityEstimation;
   private final KafkaCruiseControlConfig _config;
+  private final boolean _capacity;
```

Comment:
`_capacity` sounds like a variable storing capacity metrics. Can this be renamed to something like `_isCapacityStats`?

Reply:
Done!